valimail.com.valimail-authenticate.json

```diff
@@ -0,0 +1,38 @@
+{
+  "providerId": "valimail.com",
+  "providerName": "Valimail",
+  "serviceId": "valimail-authenticate",
+  "serviceName": "Valimail Authenticate",
+  "version": 1,
+  "logoUrl": "https://valimail-logos.s3.amazonaws.com/valimail-logo-primary.svg",
+  "description": "Enable Valimail Authenticate on a domain",
+  "variableDescription": "%spftxt% is the SPF record",
+  "records": [
+      {
+      "type": "NS",
+      "host": "_dmarc",
+      "pointsTo": "ns.vali.email",
+      "ttl": 3600
+    },
+    {
+      "type": "NS",
+      "host": "_domainkey",
+      "pointsTo": "ns.vali.email",
+      "ttl": 3600
+    },
+    {
+      "type": "NS",
+      "host": "_bimi",
+      "pointsTo": "ns.vali.email",
+      "ttl": 3600
+    },
+    {
+      "type": "TXT",
```

Comment:
For SPF please use SPFM record type.
Plain TXT record, especially with conflict mode, makes the template "own" the domains SPF, what goes beyond a scope of mail service.

Reply:
The SPF RR type has been deprecated, and must not be used, see:
- https://tools.ietf.org/html/rfc6686
- https://tools.ietf.org/html/rfc7208 (section 14.1)
- https://www.m3aawg.org/documents/en/m3aawg-best-practices-for-managing-spf-records (section 3.2.1)